### ISSUE TITLE
Shrink single image to fit screen

### ISSUE THREAD

**Issue by mus1c-123** [babydots/babybook#11]
Some of the wikipedia pics are zoomed in to fill the screen and miss key details.

**Comment by pserwylo** [comment]
Good call, I have noticed the same thing but am pondering the best way to accommodate this (and other valid requests such as your #12) without an explosion of options complicating things. At its core, I really want to maintain the simplicity of "Enter page title, get a page".

Perhaps for this specific request, it can be another option button next to the current "select image" button, which plays the part of a "Configure this image" rather than "select other images". This could have things such as "How to size the image" and maybe other options in the future too. That would mean changing the Pencil icon with something that more accurately indicates "Select from multiple images", and use the Pencil icon to instead mean "Edit this image".
